test/hotspot/jtreg/compiler/intrinsics/float16/Binary16Conversion.java

```diff
@@ -326,11 +327,12 @@ private static int testAlternativeImplementation() {
              ell++) {
             float f = Float.intBitsToFloat((int)ell);
             short s1 = Float.floatToFloat16(f);
-            short s2 =    testAltFloatToFloat16(f);
+            short s2 = testAltFloatToFloat16(f);
 
             if (s1 != s2) {
                 errors++;
-                System.out.println("Different conversion of float value " + Float.toHexString(f));
+                System.out.println("Different conversion of float value 1 (" + f + "/" + Integer.toHexString(Float.floatToRawIntBits(f)) + "): " +
+                                    Integer.toHexString(s1 & 0xffff) + "(" + s1 + ")" + " != " + Integer.toHexString(s2 & 0xffff) + "(" + s2 + ")");
```

Comment:
May be split this long line as you did in previous case.
Why you want `1` in `value 1`?

Reply:
> May be split this long line as you did in previous case.
Done

> Why you want 1 in value 1?
Oh, good catch, it's a typo, fixed.